Add slack handover automation warning (#28839)
DataDog/documentation: content/en/service_management/on-call/automations.md

### Send a Slack message
Post a message to Slack at the start or end of a shift using a handover automation. The message shows the previous and next on-call users (if any). **Note**: This requires the Datadog Slack app—make sure it's [added to your workspace][2].

#### Troubleshooting

> If you're not receiving @-mentions in Slack, it's likely your Slack and Datadog profiles aren't linked. To connect them, run any Datadog Slack command, for example `/dd page`.

Reviewer: ```suggestion
If you're not receiving @ mentions in Slack, your Slack and Datadog profiles may not be linked. To connect them, run any Datadog Slack command, such as `/dd page`.
```
